pipelines/maven-zip-build-oci-ta/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../template-build
```

Comment:
Maybe using `../maven-zip-build` as the base would be easier?

Reply:
Not sure. Let me have a check.

Reply:
Used ../maven-zip-build in new commit.